### PR TITLE
[Ubuntu] Remove apt source for podman, buildah and skopeo

### PR DESCRIPTION
**Pull request by dibir-magomedsaygitov**
# Description
In scope of this PR we are simplifying `podman`, `buildah `and `skopeo` installation on `ubuntu20`. As for `ubuntu16` and `ubuntu18` we have to use current installation approach.

## Check list
- [x] Related issue / work item is attached
- [ ] Tests are written (if applicable)
- [x] Documentation is updated (if applicable)
- [x] Changes are tested and related VM images are successfully generated

### CODE REVIEW

File: images/linux/scripts/installers/containers.sh [hunk 1]
```diff
@@ -15,4 +17,8 @@ apt-get -qq -y install ${install_packages[@]}
 mkdir -p /etc/containers
 echo -e "[registries.search]\nregistries = ['docker.io', 'quay.io']" | tee /etc/containers/registries.conf
 
+if isUbuntu20 ; then
+    rm /etc/apt/sources.list.d/devel:kubic:libcontainers:stable.list
```
Comment (miketimofeev) [inline images/linux/scripts/installers/containers.sh:21]: we need to document it somehow, not just remove from the list

File: images/linux/scripts/installers/containers.sh [hunk 2]
```diff
@@ -4,6 +4,8 @@
 ##  Desc:  Installs container tools: podman, buildah and skopeo onto the image
 ################################################################################
 
+source $HELPER_SCRIPTS/os.sh
```
Comment (maxim-lobanov) [inline images/linux/scripts/installers/containers.sh:7]: Looks like we even don't need any new repository for 20.04. See installation guide: https://podman.io/getting-started/installation

Let's get rid of it

File: images/linux/scripts/installers/containers.sh (hunk 1 above)
Comment (maxim-lobanov) [reply images/linux/scripts/installers/containers.sh:21]: Agree, May be we can collect the list of repos during image generation to some separate file like `echo "repo1" >> /image-generation/repos.txt` and attach this file to readme later?

Comment (maxim-lobanov) [reply images/linux/scripts/installers/containers.sh:21]: Btw, why we remove repo only on Ubuntu 20?

File: images/linux/scripts/installers/containers.sh (hunk 2 above)
Comment (dibir-magomedsaygitov) [reply images/linux/scripts/installers/containers.sh:7]: Sadly, but `Ubuntu20 `still requires source repo to install these tools

**Review by al-cheb — APPROVED** [review]

**Review by miketimofeev — APPROVED** [review]

**Review by dmitry-shibanov — APPROVED** [review]

**Review by maxim-lobanov — APPROVED** [review]

**Pull request merged**